app/controllers/admin/site_settings_controller.rb

```diff
@@ -8,8 +9,22 @@ def update
       params.require(:settings).map do |param|
         data = param.to_unsafe_h
         CustomSetting[data['var']] = data['value']
+        setting = CustomSetting.find_by(var: data['var']) || CustomSetting.new(var: data['var'])
+        setting.value = data['value']
+        setting.save
       end
       redirect_to '/admin/site_settings/'
     end
+
+    private
+
+      def save_current
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->